docs/lint_errors.md

```diff
@@ -83,9 +81,9 @@ The following variables throw warnings if missing:
   * The filename of the main pipeline script (recommended to be `main.nf`)
 * `timeline.file`, `trace.file`, `report.file`, `dag.file`
   * Default filenames for the timeline, trace and report
-  * Should be set to a results folder, eg: `${params.outdir}/pipeline_info/trace.[workflowname].txt"``
-  * The DAG file path should end with `.svg`
-    * If Graphviz is not installed, Nextflow will generate a `.dot` file instead
+    * Should be set to a results folder, eg: `${params.outdir}/pipeline_info/trace.[workflowname].txt"``
```

Comment:
```suggestion
  * Should be set to a results folder, eg: `${params.outdir}/pipeline_info/trace.[workflowname].txt"`
```

Still some whitespace left which breaks indentation. Also spotted this broken inline code block which was there previously.